sdk/extensions/wcf/Microsoft.CoreWCF.Azure.StorageQueues/src/Microsoft.CoreWCF.Azure.StorageQueues.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RequiredTargetFrameworks>netstandard2.0</RequiredTargetFrameworks>
```

Comment:
If you're reflecting the repository standard target, there's no reason for this override.